### PR TITLE
Add separate VMs for admin and dev tasks

### PR DESCRIPTION
**Pull request by dtufekcic**
Create separate VMs for admins and devs. Also, move all localhost ansible configuration out of the datasci playbook.

### CODE REVIEW

**Comment by dtufekcic** [comment]
Should I add a description of what these Vagrant VMs are meant for in the README?

**Comment by StarScape** [comment]
@dtufekcic I don't see why not. No one ever complained about too much documentation.

File: localadmin/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+# you're doing.
+Vagrant.configure("2") do |config|
+  # The most common configuration options are documented and commented below.
+  # For a complete reference, please see the online documentation at
+  # https://docs.vagrantup.com.
+
+  # Every Vagrant development environment requires a box. You can search for
+  # boxes at https://vagrantcloud.com/search.
+  config.vm.box = "ubuntu/bionic64"
+  config.disksize.size = "10GB"
```
Comment (StarScape) [inline localadmin/Vagrantfile:16]: Is this always going to be enough space? I imagine Ubuntu + base packages could take up several GB.

Comment (dtufekcic) [reply localadmin/Vagrantfile:16]: I think it's OK. This is the 'admin' VM so we won't be installing a lot other than Terraform and Ansible but we can bump it up.

File: configure-datasci/roles/hadoop/vars/main.yml [hunk 2]
```diff
@@ -35,6 +35,7 @@ azure_client_secret: !vault |
 # Use this selector to use a mirror for a "current" version or an "archived" one
 hadoop_mirror_selector: "current"
 hadoop_mirror:
+<<<<<<< HEAD
```
Comment (StarScape) [inline configure-datasci/roles/hadoop/vars/main.yml:38]: Looks like a leftover from a merge conflict :)

**Pull request merged**